src/components/organisms/MessagesContainer.vue

```diff
@@ -1,12 +1,13 @@
 <template>
-  <div class="flex sm:h-vh-1/2">
+  <div class="flex md:h-screen">
```

Comment:
h-screen is too big in my opinion. Maybe 3/4 would be better?

Reply:
maybe 3/4 would be better. what size do you find it too big?

Reply:
On mobile at 320px width, on landscape it looks like this: